app/assets/stylesheets/views/_organisations.scss

```diff
@@ -0,0 +1,22 @@
+.organisations {
+  .organisations__section {
+    margin-bottom: $gutter * 2;
+
+    .department-count {
+      @include bold-80;
+    }
+
+    .organisations__logo-wrapper {
+      list-style-type: none;
+      padding-bottom: $gutter-two-thirds;
+      margin-bottom: $gutter-two-thirds;
+      border-bottom: 1px solid $grey-2;
+    }
+
+    .organisations__list {
```

Comment:
Might be tempted to rename this to something more explanatory. How about `organisations__list-without-number`?